core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -166,6 +166,14 @@ oppia.factory('UrlInterpolationService', [
         return getCompleteUrl('/assets', '/images' + imagePath);
       },
 
+      /**
+       * Given an story id returns the complete url path to that image.
+       */
+      getStoryUrl: function(storyId) {
```

Comment:
Tests?

(Btw, in the future, please automatically add tests for new functionality. Otherwise the reviewer is just going to ask you for them, and this delays the whole process.)

Reply:
Added.